articles/extensions/github-deploy.md

```diff
@@ -1,52 +1,61 @@
-# GitHub Deployments
+---
+description: The GitHub Deployments extension allows you to deploy rules and database connection scripts from GitHub to Auth0. 
+---
 
-The _GitHub Deployments_ extension allows you to deploy [rules](/rules) and database connections from [GitHub](https://github.com) to Auth0. Now you can configure a GitHub repository, keep all your rules and database connection scripts there, and have them automatically deployed to Auth0 each time you push into your repository.
+# GitHub Deployments
 
-__NOTE__: This extension is currently available only for the public cloud, as extensions are not yet supported in the [appliance](/appliance).
+The **GitHub Deployments** extension allows you to deploy [rules](/rules) and database connection scripts from GitHub to Auth0. You can configure a GitHub repository, keep all your rules and database connection scripts there, and have them automatically deployed to Auth0 each time you push to your repository.
 
+__NOTE__: This extension is currently available only for the public cloud. Extensions are not yet supported in the [appliance](/appliance).
 
 ## Configure the extension
 
-To install and configure this extension, click on the __GitHub Deployments__ box in the list of provided extensions on the [Extensions](${uiURL}/#/extensions) page of the [dashboard](${uiURL}). The __Install Extension__ window pops open.
+To install and configure this extension, click on the __GitHub Deployments__ box in the list of provided extensions on the [Extensions](${uiURL}/#/extensions) page of the dashboard. The __Install Extension__ window will open.
 
 ![](/media/articles/extensions/github-deploy/install-extension.png)
 
-At this point you must set the following configuration variables:
+Set the following configuration variables:
 
-- **GITHUB_REPOSITORY**: The repository from which you want to deploy rules and database scripts. It can be either public or private. 
+- **GITHUB_REPOSITORY**: The repository from which you want to deploy rules and database scripts. This can be either a public or private repository. 
 - **GITHUB_BRANCH**: The branch that the extension will monitor for commits.
-- **GITHUB_TOKEN**: Your [personal access token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/#creating-a-token) for GitHub.
-- **SLACK_INCOMING_WEBHOOK_URL**: The Webhook URL for Slack, used in order to get Slack notifications on successful and failed deployments. Setting this value is optional.
+- **GITHUB_TOKEN**: Your GitHub personal access token. Follow the instructions at [Creating an access token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/#creating-a-token) to create a token with `admin:repo_hook` scope.
```

Comment:
@Catografix In my tests I didn't have to add this scope. I had only these set:
---

![image](https://cloud.githubusercontent.com/assets/11715799/17168358/b33f0e7e-53ea-11e6-97dc-17320ecf7ef0.png)